apiserver/facades/controller/caasapplicationprovisioner/provisioner.go

```diff
@@ -56,6 +56,12 @@ type ControllerConfigService interface {
 	WatchControllerConfig() (corewatcher.StringsWatcher, error)
 }
 
+// ModelConfigService provides access to the model configuration.
+type ModelConfigService interface {
+	ModelConfig(context.Context) (*config.Config, error)
```

Comment:
Can we add comments to these. Feel free to copy and past from the service definition. It is good to document expected behaviour.